File: Sirma Academy May 2024/JavaScript/Conditional Statements/UniversityAdmissions.js
function universityAdmissions(score, extracurriculars) {
  if (score >= 90 || (score >= 80 && score <= 89 && extracurriculars >= 2)) {
    console.log("Admitted");
  } else {
    console.log("Not admitted");
  }
}
universityAdmissions();

File: Sirma Academy May 2024/Java/Conditional Statements/src/UniversityAdmissions.java
import java.util.Scanner;

public class UniversityAdmissions {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int score = Integer.parseInt(sc.nextLine());
        int extracurriculars = Integer.parseInt(sc.nextLine());

        if(score >= 90 || score >= 80 && score <= 89 && extracurriculars >= 2){
            System.out.println("Admitted");
        } else {
            System.out.println("Not admitted");

        }
    }
}
